vtex/faststore
# Feat: Next 13 - Home - Incentives

## What's the purpose of this pull request?

Migration of the `Incentives` section from the pages router to the app router (Next 13).

## How it works?

![Screenshot 2024-08-15 at 14 55 59](https://github.com/user-attachments/assets/19794547-8d82-4c59-baba-365433e4ac97)


## How to test it?

1. Run `yarn dev`
2. Make sure that all the current pages are working as expected (`/`, `/office` and some `/<slug>/p`);
3. Make sure `/fs-next-update` is rendering with the `Incentives` section without errors.

### Starters Deploy Preview





## References

- [Jira task](https://vtex-dev.atlassian.net/browse/SFS-1200)
